sklearn/inspection/tests/test_partial_dependence.py

```diff
@@ -495,17 +495,25 @@ def test_plot_partial_dependence_multiclass(pyplot):
 
     # now with symbol labels
     target = iris.target_names[iris.target]
-    clf = GradientBoostingClassifier(n_estimators=10, random_state=1)
     clf.fit(iris.data, target)
-
-    grid_resolution = 25
     plot_partial_dependence(clf, iris.data, [0, 1],
                             target='setosa',
                             grid_resolution=grid_resolution)
-    fig = pyplot.gcf()
-    axs = fig.get_axes()
-    assert len(axs) == 2
-    assert all(ax.has_data for ax in axs)
+    fig2 = pyplot.gcf()
+    axs2 = fig2.get_axes()
+    assert len(axs2) == 2
+    assert all(ax.has_data for ax in axs2)
+
+    # check that the pd plots are the same for 0 and "setosa"
+    assert all(axs[0].lines[0]._y == axs2[0].lines[0]._y)
+    # check that the pd plots are different for another target
+    clf.fit(iris.data, iris.target)
```

Comment:
you can remove this line too ;)

Reply:
Looks like I shouldn't have removed it. That means that if I train using the targets as strings, I cannot pass an int to `plot_partial_dependence`
Don't know if that is the expected behavior or not

Reply:
Oh OK, my bad, I didn't realize it was fit on something different before

Reply:
Reverted